src/v/kafka/server/group_manager.cc

```diff
@@ -1035,14 +1035,14 @@ error_code group_manager::validate_group_status(
   const model::ntp& ntp, const group_id& group, api_key api) {
     if (!valid_group_id(group, api)) {
         vlog(
-          klog.trace, "Group name {} is invalid for operation {}", group, api);
+          klog.info, "Group name {} is invalid for operation {}", group, api);
         return error_code::invalid_group_id;
     }
 
     if (const auto it = _partitions.find(ntp); it != _partitions.end()) {
         if (!it->second->partition->is_leader()) {
             vlog(
-              klog.trace,
+              klog.info,
```

Comment:
it doesn't seem like we should log these at info level since these scenarios are part of normal operation and return to the client  clear and retrievable error codes. perhaps make it debug if trace was too noisy for some particular debug process?